prime-dotnet-webapi/Configuration/Database/Seeded Tables/LicenseConfiguration.cs

```diff
@@ -109,6 +109,47 @@ public override IEnumerable<License> SeedData
                     new License { Code = 85, Weight = 4, Name = "Non-practicing registration" },
                     new License { Code = 86, Weight = 4, Name = "Temporary registration" },
 
+
+                    //College of Oral Health Professionals
+                    // Dental Assistant
+                    new License { Code = 93, Weight = 120, Name = "Full Certified Dental Assisant" },
+                    new License { Code = 94, Weight = 121, Name = "Limited Certified Dental Assistant" },
+                    new License { Code = 95, Weight = 123, Name = "Non-Practising Certified Dental Assistant" },
+                    new License { Code = 96, Weight = 124, Name = "Temporary Certified Dental Assistant" },
+
+                    // Dental Hygienist
+                    new License { Code = 97, Weight = 131, Name = "Registered Dental Hygienist" },
+                    new License { Code = 98, Weight = 132, Name = "Dental Hygiene Practitioner" },
+                    new License { Code = 99, Weight = 133, Name = "Non-Practising Dental Hygienist" },
+                    new License { Code = 100, Weight = 134, Name = "Temporary Dental Hygienist" },
+
+                    // Dental Technician
+                    new License { Code = 101, Weight = 141, Name = "Dental Technician" },
+                    new License { Code = 102, Weight = 142, Name = "Student Dental Technician" },
+                    new License { Code = 103, Weight = 143, Name = "Non-Practising Dental Technician" },
+                    new License { Code = 104, Weight = 144, Name = "Temporary Dental Technician" },
+
+                    // Dental Therapist
+                    new License { Code = 105, Weight = 110, Name = "Dental Therapist" },
+
+                    // Dentist
+                    new License { Code = 106, Weight = 100, Name = "Full Dentist" },
+                    new License { Code = 107, Weight = 102, Name = "Limited (Academic) Dentist" },
+                    new License { Code = 108, Weight = 103, Name = "Limited (Armed Services or Government) Dentist" },
+                    new License { Code = 109, Weight = 104, Name = "Limited (Education & Volunteer) Dentist" },
+                    new License { Code = 110, Weight = 105, Name = "Limited (Restricted-to-Specialty) Dentist" },
+                    new License { Code = 111, Weight = 106, Name = "Student Dentist" },
+                    new License { Code = 112, Weight = 107, Name = "Non-Practising Dentist" },
+                    new License { Code = 113, Weight = 108, Name = "Temporary Dentist" },
+
+                    // Denturist
```

Comment:
Please check ordering of licence classes versus https://oralhealthbc.ca/register-renew/registrant-classifications/denturist/

Reply:
The last 3 licence classes have been re-ordered from Leanne in the excel spreadsheet. It is different that the order in web site.